keep/contextmanager/contextmanager.py

```diff
@@ -185,9 +185,7 @@ def set_step_context(self, step_id, results, foreach=False):
         self.steps_context_size = asizeof(self.steps_context)
 
     def get_last_workflow_run(self, workflow_id):
-        # TODO: fix for throttling
-        # no previous runs
-        return {}
+        return get_last_workflow_execution_by_workflow_id(workflow_id, self.tenant_id)
```

Comment:
should follow convention of tenant_id as first parameter

Reply:
that function not in my changeset originally but I'm gonna update it as well.

Reply:
done